qlib/SoapClient.qm

```diff
@@ -38,7 +38,7 @@
 %new-style
 
 module SoapClient {
-    version = "0.2.4.1";
```

Comment:
the version should be 0.2.5 - 0.2.5 has not been released yet - 0.2.4.* was for the release that went with Qore 0.8.12.*